index.html

```diff
@@ -0,0 +1,666 @@
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+        </section>
+    </div>
+
+        <div class="article2">
+            <ul class="article2-sub1 orange-headers">
+                <li>Latest funding rounds on Index.co
+                    <i class="fas fa-angle-right"></i>
+                </li>
+            </ul>
+
+            <ul class="article2-subelement">
+                <li class="one">Add info</li>
+                <li class="two">Get Update by email</li>
+                <li class="three">see more</li>
+            </ul>
```

Comment:
- [ ] On the medium screen size, it should have 3 items per row

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/88700543-3b560900-d0df-11ea-9e3c-27dae01f28b8.png) | ![image](https://user-images.githubusercontent.com/61479648/88700571-4446da80-d0df-11ea-8f88-e92f165dabc5.png)|

- [ ] On the mobile is too different. Try to make it closer to the original. (there is an image missing on the right)

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88700711-735d4c00-d0df-11ea-85be-c30f2acb9c38.png) | ![image](https://user-images.githubusercontent.com/61479648/88700752-7ce6b400-d0df-11ea-8a00-4b77a53682ea.png) |